content/blog/automate-boot-from-san-target-configurations-using-redfish.md

```diff
@@ -0,0 +1,277 @@
+
+`ilorest rawget /redfish/v1/Chassis/1/NetworkAdapters/`
+
+The previous command returns the following output: 
+
+`{
+  "@odata.context": "/redfish/v1/$metadata#NetworkAdapterCollection.NetworkAdapterCollection",
+  "@odata.etag": "W/"F09DE05C"",
+  "@odata.id": "/redfish/v1/Chassis/1/NetworkAdapters/",
+  "@odata.type": "#NetworkAdapterCollection.NetworkAdapterCollection",
+  "Description": "The collection of network adapter resource instances available in this chassis.",`
+
+`"Members": [
+    {
+      "@odata.id": "/redfish/v1/Chassis/1/NetworkAdapters/DC080000/"
+    },
+    {
+      "@odata.id": "/redfish/v1/Chassis/1/NetworkAdapters/DE083000/"
+    },
+    {
+      "@odata.id": "/redfish/v1/Chassis/1/NetworkAdapters/DE081000/"
+    },
+    {
+      "@odata.id": "/redfish/v1/Chassis/1/NetworkAdapters/DE07B000/"
+    }
+  ],`
+
+`"Members@odata.count": 4,
+…`
+
+Make a note of the device IDs highlighted in the output above.  The values for these highlighted items might be different on the server you are using.
+
+Query these items using the device ID to verify it is the right FC card that you would like to configure, for example using DE083000 device (your output and device IDs might be different).
+
+`ilorest rawget /redfish/v1/Chassis/1/NetworkAdapters/DE083000`
+
+Look for the following in the output.
+
+`"Location": {
+        "PartLocation": {
+          "LocationOrdinalValue": 4,
+          "LocationType": "Slot",
+          "ServiceLabel": "PCI-E Slot 4"
+        }
+…`
+
+`"Manufacturer": "Hewlett Packard Enterprise",`
+
+`"Model": "HPE StoreFabric SN1610E 32Gb 2-port Fibre Channel Host Bus Adapt",`
+
+`"Name": "HPE SN1610E 32Gb 2p FC HBA"`
+
+The above values can help you ascertain that you have the right card, and its device ID.
+
+## Step 3: Verify that Redfish configuration is Enabled
+
+Before you can change the boot from SAN configurations on an FC card, you need to have a variable named "RedfishConfiguration" set to "Enabled" for the specific adapter. 
+
+The current value of this variable can be viewed by querying the adapter.
+
+`ilorest rawget /redfish/v1/Chassis/1/NetworkAdapters/DE083000`
+
+Towards the end of the generated output from the above command, please check for an entry. 
+"RedfishConfiguration": "Enabled"
+
+If this value is not set, please perform  the following:
+
+1. Create a file named enable-redfish.txt with the following contents
+
+{
+
+  "/redfish/v1/Chassis/1/NetworkAdapters/DE083000/":
+
+ {
+
+  "Oem":
+
+  {
+
+"Hpe": {
+
+"RedfishConfiguration": "Enabled"
+
+}
+
+}
+
+}
+
+}
+
+2 Apply this patch via iLOrest
+`ilorest rawpatch <file path>\enable-redfish.txt`
+
+`3﻿.`To make this patch permanent, flush the configuration to the iLO NVRAM by creating a file name flushtonvm.txt, with the following contents
+`{
+  "/redfish/v1/Chassis/1/NetworkAdapters/DE083000/Actions/Oem/Hpe/HpeNetworkAdapter.FlushConfigurationToNVM/":
+  {`\
+`}
+}`
+
+4 Post the flush action via iLOrest
```

Comment:
suggest to use the bullet list numbere: 
4. Post the flush action via iLOrest
5. Reboot the server